nixos/modules/services/networking/shairport-sync.nix

```
@@ -36,6 +36,14 @@ in
         '';
       };
 
+      openFirewall = mkOption {
+        type = types.bool;
+        default = true;
```

```suggestion
        default = false;
```

Oh, sorry, I missed this first time around. We have a policy where the `default` value here **must** be `false` for every `openFirewall` option except for `services.ssh.openFirewall`.

---

Ah, didn't know that was the case.  I've now changed the default to `false`.  Thanks for the review!